test/RoadRegistry.Tests/BackOffice/Uploads/RecordTypeTests.cs

```diff
@@ -103,6 +104,26 @@ public void AllReturnsExpectedResult()
                 RecordType.All);
         }
 
+        [Fact]
+        public void IsAnyOfReturnsExpectedResultIfItIsAnyOfThese()
```

Comment:
Useful to verify if a record type is one of the allowed / supported record types ...